nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-processors/src/main/java/org/apache/nifi/processors/script/InvokeScriptedProcessor.java

```diff
@@ -92,11 +92,10 @@
                     logger.error(message, t);
                 }
             }
-        } else {
-            // Return defaults for now
-            relationships.add(REL_SUCCESS);
-            relationships.add(REL_FAILURE);
         }
+        // Add defaults
+        relationships.add(REL_SUCCESS);
+        relationships.add(REL_FAILURE);
```

Comment:
@mattyb149 do you mind having a look? I don't think this is what we want here, I'd rather say that if we need to change something it would be on tests side.